symbols/overlay29.yml

```diff
@@ -384,6 +395,22 @@ overlay29:
         Has a special check to return false for Labyrinth Cave B10F (the Gabite boss fight).
         
         return: bool
+    - name: CheckSlowStart
```

Comment:
I don't really like calling this "Check*" if it's actually doing something. Let's call it `TryActivateSlowStart`